docs/translation/README.md

```diff
@@ -0,0 +1,91 @@
+<img src="../../public/favicon.png" width="64" align="left">
+
+# Guidelines to translations in Strype
+
+The process for translating Strype is simple. It relies on text files, and the reference for all translations is the English version.
+
+This guide explains:
+
+-   [How to retrieve or create the translation files](#how-to-retrieve-or-create-the-translation-filess)
+-   [How to understand the translations files](#how-to-understand-the-translations-files)
+-   [How to do the translations.](#how-to-do-the-translations)
+-   [How to visualise the translations on a test project.](#how-to-visualise-the-translations-on-a-test-project)
+-   [How to submit the translations.](#how-to-submit-the-translations)
+
+This guide focuses on translations *directly* handled by the Strype website. Translators may otherwise contribute to the localisation of <a href="https://github.com/Tobias-Kohn/TigerPython-Parser" target="_blank">TigerPython</a>, which is used by Strype to display Python parsing error messages.
+
+>[!NOTE]
+>Currently, all translations are taking effect in Strype when a new version of the application is released by the Strype team. So even if a translation has been received by the Strype team and included into the source code repository, it will be visible in the online tool only when the Strype team releases a new version.
+
+## How to retrieve or create the translation files
+
+To retrieve the translation files, you will need to locate them inside the Strype source code repository hosted in GitHub (<a href="https://github.com/k-pet-group/Strype" target="_blank">https://github.com/k-pet-group/Strype</a>).
+
+The translation files for all supported locales are inside the `/src/localisation/` folder, which contains **one subfolder per locale (as a 2-letter code)**. For example, the translations for English are in <code>/src/localisation/**en**/</code>, those of Spanish are in <code>/src/localisation/**es**/</code> and so on.
```

Comment:
Might be worth linking to this page https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes which lists the two letter codes (in case they don't know the code for the new language).

Reply:
Good idea.